client/my-sites/stats/activity-log-item/index.jsx

```diff
@@ -310,6 +313,7 @@ class ActivityLogItem extends Component {
 	renderSummary() {
 		const {
 			allowRestore,
```

Comment:
It is unfortunate that we have these similarly named props, though. Perhaps we should pass around `inProgress` instead of `disableRestore`?